frontend/packages/ux-editor/src/hooks/mutations/useAddGroupMutation.ts

```diff
@@ -34,7 +34,7 @@ const getNextPageNumber = (groups: GroupModel[], t: (key: string) => string): nu
     ?.flatMap((group) => group.order?.map((page) => page.id) || [])
     .reduce((max, id) => {
       const match = id?.match(new RegExp(`${t('general.page')}(\\d+)`));
-      return match ? Math.max(max, parseInt(match[1], 10)) : max;
+      return match && !isNaN(parseInt(match[1], 10)) ? Math.max(max, parseInt(match[1], 10)) : max;
```

Comment:
I believe `parseInt` is in base 10 by default, so we don't need to explicitly add this.
```suggestion
      return match && !isNaN(parseInt(match[1])) ? Math.max(max, parseInt(match[1])) : max;
```